qa/rpc-tests/wallet_persistence.py

```diff
@@ -33,9 +34,23 @@ def setup_network(self, split=False):
         self.sync_all()
 
     def run_test(self):
+        # Slow start is not enabled for regtest, so the expected subsidy starts at the
+        # maximum, but the hardcoded genesis block for regtest does not consume the
+        # available subsidy.
+        pre_halving_blocks = 143
+        pre_halving_subsidy = Decimal('12.5')
+        post_halving_blocks = 57
+        post_halving_subsidy = Decimal('6.25')
+        expected_supply = \
+            pre_halving_blocks * pre_halving_subsidy + \
+            post_halving_blocks * post_halving_subsidy
```

Comment:
Non-blocking nit: I prefer this style.
```suggestion
        expected_supply = (pre_halving_blocks * pre_halving_subsidy +
                           post_halving_blocks * post_halving_subsidy)
```